packages/vite/src/node/ssr/runtime/moduleCache.ts

```diff
@@ -50,7 +50,7 @@ export class ModuleCacheMap extends Map<string, ModuleCache> {
         importers: new Set(),
       })
     }
-    return mod as ModuleCache
+    return mod
```

Comment:
This `as ModuleCache` is not needed because `super.get(modulePath)!` is already `ModuleCache` type.